index.js

```diff
@@ -145,6 +170,17 @@ function watchify (b, opts) {
 
     b.on('transform', function (tr, mfile) {
         tr.on('file', function (dep) {
+            cache._transformDeps[mfile] = cache._transformDeps[mfile] || [];
+            cache._transformDeps[mfile].push(dep);
```

Comment:
Should we check for uniqueness of the `dep` in the array here? Incremental changes while the process is still running will keep appending the same deps.